src/styles/output.css

```
@@ -1474,338 +911,6784 @@
+  backdrop-filter: blur(10px);
+}
+.send-success-dialog {
+  width: min(620px, 100%);
+  display: grid;
+  gap: 18px;
+  padding: 28px;
+  border: 1px solid color-mix(in srgb, #2fd483 34%, #272a33);
+  @supports (color: color-mix(in lab, red, red)) {
+    border: 1px solid color-mix(in srgb, var(--color-success) 34%, var(--color-line));
+  }
+  border-radius: 12px;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)